package-lock.json

```diff
@@ -16,7 +16,7 @@
         "mustache": "^4.2.0",
         "post-me": "^0.4.5",
         "punycode": "^2.1.1",
-        "skynet-js": "^4.0.18-beta",
+        "skynet-js": "^4.0.19-beta",
```

Comment:
this is still outdated, should be updated after we update `skynet-js` by simply running `npm i`